packages/strapi-plugin-settings-manager/admin/src/containers/Home/index.js

```diff
@@ -179,18 +243,95 @@ export class Home extends React.Component { // eslint-disable-line react/prefer-
 
   renderListButtonLabel = () => `list.${this.props.params.slug}.button.label`;
 
+  renderPopUpFormLanguage = (section, props) => (
+    map(section.items, (item, key) => (
+      <div key={key}>
+        <form>
+          {props.renderInput(item, key)}
+        </form>
+      </div>
+    ))
+  )
+
+  renderRowDatabase = (props, key, listStyles) => {
+    let provider;
+
+    // TODO provider
+    switch(props.provider) {
```

Comment:
Agree on that one that's why there is a // TODO comment.
This feature is not fully implemented plus I also need the color rules.